src/Hangman.java

```diff
@@ -0,0 +1,82 @@
+import java.util.List;
+
+public class Hangman {
+
+    private List<Character> characters;
+    private List<Character> coveredWord;
+    private List<Character> copyCharacters;
+
+    public Hangman(List<Character> characters, List<Character> coveredWord, List<Character> copyCharacters) {
+        this.characters = characters;
+        this.coveredWord = coveredWord;
+        this.copyCharacters = copyCharacters;
+    }
+
+    public List<Character> getCharacters() {
+        return characters;
+    }
+
+    public void setCharacters(List<Character> characters) {
+        this.characters = characters;
+    }
+
+    public List<Character> getCoveredWord() {
+        return coveredWord;
+    }
+
+    public void setCoveredWord(List<Character> coveredWord) {
+        this.coveredWord = coveredWord;
+    }
+
+    public List<Character> getCopyCharacters() {
+        return copyCharacters;
+    }
+
+    public void setCopyCharacters(List<Character> copyCharacters) {
+        this.copyCharacters = copyCharacters;
+    }
+
+    public void changeStringToCharactersList(String randomWord) {
+        for (char c : randomWord.toCharArray()) {
```

Comment:
I think this can be done without for loop. Just use utility method to convert array to list. Arrays.asList(randomWord.toCharArray);

Reply:
That method will create List<char[]> and i need List<Characters>, also i have there two list to witch i add chars.